hack/lib/images.bash

```diff
@@ -3,16 +3,25 @@
 # shellcheck disable=SC1091,SC1090
 source "$(dirname "${BASH_SOURCE[0]}")/metadata.bash"
 
-registry_host='registry.ci.openshift.org'
-registry="${registry_host}/openshift"
+ci_registry_host='registry.ci.openshift.org'
+ci_registry="${ci_registry_host}/openshift"
+
 export CURRENT_VERSION_IMAGES=${CURRENT_VERSION_IMAGES:-"main"}
+CURRENT_VERSION="$(metadata.get project.version)"
+
+quay_registry_app_version=${CURRENT_VERSION/./} # 1.34.0 -> 134.0
+quay_registry_app_version=${quay_registry_app_version%.*} # 134.0 -> 134
+registry_host="quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-release-${quay_registry_app_version}"
```

Comment:
The current issue with this is that we wouldn't respect the per-component configuration

Reply:
Resolved in the last commit